src/Modifier/Product/VariantAttributesModifier.php

```diff
@@ -7,9 +7,11 @@
 use Doctrine\DBAL\Driver\Result;
 use Doctrine\DBAL\Exception as DBALException;
 use Makaira\OxidConnectEssential\Modifier;
+use Makaira\OxidConnectEssential\SymfonyContainerTrait;
```

Comment:
This mustn't be used here. It is only for overwritten OXID classes where dependency injection is not possible.

```suggestion
```